Source/Core/DolphinQt2/MainWindow.cpp

```diff
@@ -316,11 +319,13 @@ void MainWindow::Pause()
 
 bool MainWindow::Stop()
 {
-  if (!Core::IsRunning())
+  if (!Core::IsRunning() || m_inquiry_in_progress)
```

Comment:
Hmm this makes me uneasy, since it's duplicating what Qt is supposedly already doing and we don't know why it's needed/how to reproduce the issue. This could very easily be removed in some future cleanup. @spycrab, @mimimi085181, can one of you figure out how to reproduce the issue?